srcpkgs/xbps-triggers/files/dkms

```diff
@@ -21,19 +21,30 @@ UPDATE="$5"
 export PATH="usr/bin:usr/sbin:/usr/sbin:/usr/bin:/sbin:/bin"
 
 remove_modules() {
-	local _modver _kver
+	local _modver _kver _line
 
 	# Remove the specified modules from all kernels.
 	set -- ${dkms_modules}
 	while [ $# -gt 0 ]; do
-		$DKMS status -m "$1" | while read line; do
-			if $(echo "$line" | egrep -vq '(added|built|installed)'); then
+		$DKMS status -m "$1" | while IFS=' ' read _modver _kver _line; do
+			if echo "$_modver $_kver $_line" | grep -Evq '(added|built|installed)'; then
 				shift 2; continue
 			fi
-			_modver=$(echo "$line"|sed "s/$1,[[:blank:]]\([^,]*\)[,:].*/\1/;t;d")
-			_kver=$(echo "$line"|awk '{print $3}'|sed "s/\(.*\),$/\1/")
-			echo -n "Removing DKMS module '${1}-${_modver}' for kernel-${_kver}... "
-			$DKMS remove -m "$1" -v "${_modver}" -k "${_kver}" >/dev/null 2>&1
+			_modver=${_modver##*/}
+			_kver=${_kver%,}
```

Comment:
This breaks version parsing for old versions of DKMS, which we can't do in case somebody does a partial system update. Suggest instead:
```sh
$DKMS status -m "$1" | while IFS="/, " read _modname _modver _line; do
	_modver="${_modver%:}"
	shift 2
```
which *should* reliably separate `$_modname` and `$_modver` in both old and new DKMS, leaving the rest of the line intact. Note that we still strip the possible `:` suffix in the version here.